docs/documentation/features/rules.md

```diff
@@ -30,21 +35,21 @@ Main building blocks of the rules are modules and each rule consists of one or m
     condition - which acts like a filter for the trigger event and can evaluate the event properties or the state of the system;
```

Comment:
should be more general here: "which acts like a filter depending on the defined condition type and its input and configuration. For example evaluation of trigger outputs or item states."
Also the trigger should be described in a more general way. The eventTrigger is only one example for a trigger.